climada/entity/exposures/litpop/litpop.py

```diff
@@ -0,0 +1,1108 @@
+            * 'nfw': non-financial wealth (Source: Credit Suisse, of households only)
+            * 'tw': total wealth (Source: Credit Suisse, of households only)
+            * 'norm': normalized by country
+            * 'none': LitPop per pixel is returned unchanged
+            The default is 'pc'.
+        total_values : list contaiuning numerics, same length as countries (optional)
+            Total values to be disaggregated to grid in each country.
+            The default is None. If None, the total number is extracted from other
+            sources depending on the value of fin_mode.
+        admin1_calc : boolean, optional
+            If True, distribute admin1-level GDP (if available). Default: False
+        reference_year : int, optional
+            Reference year for data sources. Default: 2020
+        gpw_version : int (optional)
+            Version number of GPW population data.
+            The default is None. If None, the default is set in gpw_population module.
+        data_dir : Path (optional)
+            redefines path to input data directory. The default is SYSTEM_DIR.
+        resample_first : boolean
+            First resample nightlight (Lit) and population (Pop) data to target
+            resolution before combining them as Lit^m * Pop^n?
+            The default is True. Warning: Setting this to False affects the
+            disaggregation results - expert choice only
+
+        Raises
+        ------
+        ValueError
+        """
+        if isinstance(countries, str) or isinstance(countries, int):
+            countries = [countries] # for backward compatibility
+
+        # set res_arcsec if not given:
+        if res_arcsec is None:
+            if res_km is None:
+                res_arcsec = 30
+            else:
+                res_arcsec = 30 * res_km
+            LOGGER.info('Resolution is set to %i arcsec.', res_arcsec)
+        if fin_mode is None:
+            fin_mode = 'pc'
+        # value_unit is set depending on fin_mode:
+        if fin_mode in ['none', 'norm']:
+            value_unit = ''
+        elif fin_mode == 'pop':
+            value_unit='people'
+        else:
+            value_unit = 'USD'
+        if total_values is None:
+            total_values = [None] * len(countries)
+        elif len(total_values) != len(countries):
+            raise ValueError("'countries' and 'total_values' must be lists of same length")
+        tag = Tag()
+
+        # litpop_list, a list containing one Exposure instance per country and
+        # None for countries that could not be identified is produced:
+        if admin1_calc: # each admin 1 region initiated seperately,
+                        # with total value share based on subnational GDP share.
+                        # This requires GRP (Gross Regional Product) data in the
+                        # GSDP data folder.
+            litpop_list = [_calc_admin1(country, res_arcsec, exponents, fin_mode,
```

Comment:
For consistency, it should maybe be 
```suggestion
            litpop_list = [_set_admin1_one_country(country, res_arcsec, exponents, fin_mode,
```